do/loadbalancers.go

```diff
@@ -251,6 +280,31 @@ func (l *loadbalancers) buildLoadBalancerRequest(service *v1.Service, nodes []*v
 	}, nil
 }
 
+func (l *loadbalancers) waitActive(lbID string) (*godo.LoadBalancer, error) {
+
+	ctx, cancel := goctx.WithTimeout(goctx.TODO(), time.Second*time.Duration(l.lbActiveTimeout))
+	defer cancel()
+	ticker := time.NewTicker(time.Second * time.Duration(l.lbActiveCheckTick))
```

Comment:
You should call `defer ticker.Stop()` otherwise it'll leak goroutines.

Reply:
doh!